src/modules/overview/api/auctionType.ts

```diff
@@ -2,3 +2,8 @@ export enum AuctionType {
   FixedSwap = 'fixed-swap',
   EnglishAuction = 'english-auction',
 }
+
+export enum AuctionType_number {
```

Comment:
No way to do it like this, let's discuss it online

All the interfaces must be unified to AuctionType in mappers

Reply:
And [here](https://github.com/bouncefinance/bounce-market/blob/stage/src/modules/common/api/poolType.ts#L3) is already created the same types